hack/e2e/eksctl/eksctl.sh

```diff
@@ -43,6 +43,7 @@ function eksctl_create_cluster() {
     K8S_VERSION="${K8S_VERSION}" \
     ZONES="${ZONES}" \
     INSTANCE_TYPE="${INSTANCE_TYPE}" \
+    BOTTLEROCKET="${BOTTLEROCKET:-false}" \
     WINDOWS="${WINDOWS}" \
```

Comment:
Suggest setting the default value in `hack/e2e/config.sh` for consistency:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/ba7a117a614d1c61a2a49ab72cae3e1626f30dcb/hack/e2e/config.sh#L32